### ISSUE TITLE
CTRL + D deselects an image when it shouldn't and crashes it position

### ISSUE THREAD

**Issue by abner-magalhaes** [photopea/photopea#5491]
How to reproduce:

1. Import an PNG image with blank spaces;
2. Press CTRL + ALT + T to change the size, but don't change;
3. Press CTRL + D (deselect) and see that positions (x or y) changes;

[Here is a example video](https://youtu.be/uUyjEPlW-lY).

**Comment by photopea** [comment]
I am afraid I dont understand. You have no selections and Ctrl+D just escapes the Free Transform tool.

I think you are confused, before Transform Controls of the Move tool are around non-transparent pixels, while the Free Transform is around the original image (can be rotated). That is how it works in Adobe Photoshop and I am a bit afraid to change it, as many of our users are users of Photoshop.

**Comment by abner-magalhaes** [comment]
Hello! Thanks for such a quick response!

I found it strange the fact that I was able to execute a command that shouldn't be executed at this moment, in this case CTRL + D. I realized that this occurs in a Smart Object.

I press CTRL + T to perform the transformation on an object. In the "Properties->Layer" window, information such as width, height, x-position and y-position appears. Until then, everything is fine. But if I press CTRL + D, the object's xy positioning information changes. But, if I press CTRL + T again, the information changes again. This leaves me extremely confused, what is the true information anyway?

Try to notice in the video that CTRL+T ---> CTRL+D ---> CTRL+T displays different information about the XY position.

Is this expected?
